pkg/geo/geo.go

```diff
@@ -305,6 +305,11 @@ func (g *Geometry) ShapeType() geopb.ShapeType {
 	return g.spatialObject.ShapeType
 }
 
+// ShapeType returns the 2D shape type of the Geometry.
+func (g *Geometry) ShapeType2D() geopb.ShapeType {
+	return g.spatialObject.ShapeType.To2D()
```

Comment:
Is there a reason you chose not to use `g.ShapeType().To2D()`?

Reply:
Nothing in particular - just copy paste. Doesn't matter too much, heh.